### ISSUE TITLE
Chronicle test data should create visits over multiple days

### ISSUE THREAD

**Issue by pdehaan** [mozilla/chronicle#345]
Found during testing #341.
From [/bin/create_test_data.js:35-36](https://github.com/mozilla/chronicle/blob/6f3b6489250b177fe68cebdca33691a9a333939e/bin/create_test_data.js#L35-36):

> _We'll use this date as a starting point for generating records. Each successive record will be a number of seconds in the future._

> ```js
visitedAt: new Date(historyDate.getTime() + (1000 * 60 * n)).toJSON()
```

Basically all the test results are loaded into the same day. We should probably bump it up to a handful of results for each day so we can test headers and deleting a bit easier.

**Comment by pdehaan** [comment]
Attempted fix made in #346

**Comment by 6a68** [comment]
I wonder if it would be good to extract the dates out into the json file that contains the visit URLs, so that we can easily experiment with, say, UI for users who have been gone for a few weeks, then return.

I'll leave that as something to think about for later.
